docs/gpus.rst

```diff
@@ -53,6 +53,12 @@ by installing an `nv_peer_memory <https://github.com/Mellanox/nv_peer_memory>`__
    .. code-block:: bash
 
        $ HOROVOD_GPU_ALLREDUCE=NCCL pip install --no-cache-dir horovod
+   
+   If you have installed NCCL 2 using the CentOS / RHEL package, you can run:
```

Comment:
have installed --> installed

Reply:
Let's align the sentence above with the same.

Reply:
@alsrgv @sblotner do you suggest to correct the grammar in all 5 places with "present perfect tense" in this document?

Reply:
done!